packages/atlas/src/components/_auth/AuthenticationModalStepTemplate/AuthenticationModalStepTemplate.tsx

```diff
@@ -57,9 +59,12 @@ export const AuthenticationModalStepTemplate: FC<AuthenticationModalStepTemplate
             </Text>
           </HeaderContainer>
           {subtitle && (
-            <Text variant="t200" as="span" color="colorText">
-              {subtitle}
-            </Text>
+            <>
+              {subtitleIcon}
+              <Text style={{ whiteSpace: 'pre-line' }} variant="t200" as="span" color="colorText">
```

Comment:
Inline style

Reply:
actually we don't even need it here since I changed subtitle to be from plain text to the reactNode